compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -1,16 +1,28 @@
+// #![deny(rustc::untranslatable_diagnostic)]
+// #![deny(rustc::diagnostic_outside_of_impl)]
```

Comment:
Could you remove the commented out code? I can just approve the code when it doesn't contain comments like this and is otherwise okay.